painttiming.bs

```diff
@@ -27,6 +27,30 @@ urlPrefix: https://www.w3.org/TR/hr-time-2/#idl-def-domhighrestimestamp; spec: H
     type: typedef; text: DOMHighResTimeStamp
 urlPrefix: https://www.w3.org/TR/CSS2/visuren.html; spec: CSS-2;
     type: dfn; url: #viewport; text: viewport
+urlPrefix: https://www.w3.org/TR/CSS22/visufx.html; spec: CSS-2;
```

Comment:
Did you check that all of these are actually needed? Bikeshed should be able to grab references when they're properly exported.

Reply:
I didn't check, I can try removing some of them and see what bs comes up with.

Reply:
I removed some of the references that bikeshed could  automatically import.

Reply:
@npm1 is there a quick way to find in bikeshed whether some of the existing refs can be removed? I found that searching github for this is tedious...

Reply:
You can remove the link from the anchors, run bikeshed spec locally, and see if it warns you about missing refs.

Reply:
OK that's what I've done